src/components/more-blogs-section/more-blogs-section.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+
+import { Link } from "gatsby";
+import PropTypes from "prop-types";
+
+import { classnames } from "../../helpers";
+
+import { Card } from "../card";
+
+import * as styles from './more-blogs-section.module.scss';
+
+const MoreBlogsSection = ({ data, refMoreFrom, title }) => {
+  const { edges } = data.allMdx;
+
+  return (
+    <div className={styles.moreFromXlContainer} ref={refMoreFrom} >
+      <div className={styles.titleContainer}>
+        <Link to="/" className={classnames("text__heading__three__blueTwo", styles.titleStyle)}>{`${title} →`}</Link>
+      </div>
+      <div className={styles.blogsContainer}>
+        {edges.map(({node}) => <Card data={node} key={node.id} className={styles.cardStyles} />)}
+        {console.log(refMoreFrom)}
+      </div>
+    </div>
+  );
+}
+
+export { MoreBlogsSection };
+
+MoreBlogsSection.propTypes = {
+  refMoreFrom: PropTypes.object,
+  title: PropTypes.string,
+  refMoreFrom: PropTypes.shape({
+    current: PropTypes.null,
```

Comment:
what does PropTypes.null mean?

Reply:
I have tried to apply HTML Element, but it gives error in Gatsby, but, if I write null that error does not occur.
Null is the absence of a value, current can have different values.

Reply:
I prefer that you use `object` rather than `null`

Reply:
done